src/App.test.js

```diff
@@ -1,8 +1,16 @@
-import { render, screen } from '@testing-library/react';
+import { screen, render, waitForElementToBeRemoved } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('renders page title', async () => {
   render(<App />);
```

Comment:
If you want to look for loading, I think a better way would be to await for it as a heading element and shy away from the waitForElementToBeRemoved. Or you can ignore loading, if what you really want is to find a specific element. Just await/findBy the element you want.